# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: modules/libpref/init/StaticPrefList.yaml
# Commit: 8cceecf2808b
# Full Hash: 8cceecf2808be894589d0d05c31934114b6a65d2
# Author: Andreas Pehrson <apehrson@mozilla.com>
# Date: 2022-12-20 16:15:59
# Description:
#   Bug 1806521 - Disable new mac camera backend. r=karlt
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D165124
# ==============================================================================

diff -r b47bdbd89fd4 -r 8cceecf2808b modules/libpref/init/StaticPrefList.yaml
--- a/modules/libpref/init/StaticPrefList.yaml	Tue Dec 20 07:31:25 2022 +0000
+++ b/modules/libpref/init/StaticPrefList.yaml	Tue Dec 20 07:32:06 2022 +0000
@@ -10390,7 +10390,7 @@
 # disabled, an older forked capture module is used.
 - name: media.getusermedia.camera.macavf.enabled
   type: bool
-  value: @IS_EARLY_BETA_OR_EARLIER@
+  value: false
   mirror: once
 
 # WebRTC prefs follow
